tests/sys/psa_crypto/main.c

```diff
@@ -73,7 +74,9 @@ static void test_hash_interleaved(void)
 {
     const psa_algorithm_t alg = PSA_ALG_SHA_256;
 
+    NONSTRING
     const uint8_t in1[1] = "a";
+    NONSTRING
     const uint8_t in2[1] = "b";
```

Comment:
look like these should be 
``` c
in1[1] = 'a';
in2[1] = 'b';
```

Reply:
No, that is invalid syntax.

You could do

```C
const uint8_t in1[1] = { 'a' };
```

instead. But that would be the same result as before.